scripts/openroad/ioplacer.tcl

```diff
@@ -37,14 +36,17 @@ ppl::set_hor_thick_multiplier $::env(FP_IO_HTHICKNESS_MULT)
 
 set opts ""
 if { $::env(FP_IO_MODE) == 1 } {
-    set opts "-random"
+	set opts "-random"
+}
+
+if { $::env(FP_IO_MIN_DISTANCE) != "" } {
+	append opts "-min_distance $::env(FP_IO_MIN_DISTANCE)"
```

Comment:
You are missing a space inside the initial quote

Reply:
I'm just going to replace it with a lappend